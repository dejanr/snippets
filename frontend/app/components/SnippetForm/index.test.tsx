import { expect, test } from 'vitest'
import { render } from '@testing-library/react'
import { SnippetForm } from './index'

test('it renders', () => {
  const { container } = render(<SnippetForm />)

  expect(container.firstChild).toMatchSnapshot()
})
