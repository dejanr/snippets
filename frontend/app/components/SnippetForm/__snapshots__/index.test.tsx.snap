// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`it renders 1`] = `
<form
  class="mx-full mb-4"
>
  <div
    class="mb-4"
  >
    <textarea
      class="w-full px-3 py-2 border border-gray-300 focus:outline-none focus:ring-2 focus:ring-blue-500"
      id="text"
      name="text"
      placeholder="Enter your code snippet here..."
      required=""
      rows="8"
    />
  </div>
  <button
    class="w-full bg-blue-600 py-2 px-4 hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-500"
    type="submit"
  >
    <span
      class="text-white"
    >
      Create Snippet
    </span>
  </button>
</form>
`;
